feature/topics/src/main/kotlin/uk/govuk/app/topics/data/TopicsRepo.kt

```
@@ -17,6 +18,11 @@ internal class TopicsRepo @Inject constructor(
     private val localDataSource: TopicsLocalDataSource,
 ) {
 
+    private lateinit var _stepBySteps: List<RemoteTopicContent>
+
+    val stepBySteps
+        get() = _stepBySteps
```

Could this be open to an exception?

---

Yes but IMO that would be preferable to making it nullable and having to handle the null. The state in which it would be null or uninitialised would be a dev error i.e. trying to access it before fetching a topic so my thinking is better to fail fast and flag the issue rather than fail silently.